apps/domain/src/domain/domain.controller.ts

```
@@ -21,120 +19,111 @@ import { ProviderHealthService } from "../providers/llm/provider-health.service"
-  async streamPrompt(
+  async prompt(
     @Body() dto: LlmPromptRequestDto,
     @Res({ passthrough: false }) response: FastifyReply,
     @Headers("Last-Event-ID") lastEventId?: string,
-  ): Promise<void> {
-    // Validate that stream flag is set for streaming endpoint
-    if (!dto.stream) {
-      throw new HttpException(
-        "Stream flag must be true for streaming endpoint",
-        HttpStatus.BAD_REQUEST,
+  ): Promise<LlmResponse | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)